lib/matplotlib/patches.py

```diff
@@ -4057,7 +4057,7 @@ def __init__(self, posA=None, posB=None,
     def set_dpi_cor(self, dpi_cor):
         """
         dpi_cor is currently used for linewidth-related things and
-        shrink factor. Mutation scale is not affected by this.
```

Comment:
just drop the 'not', in the near future no one will remember the old behavior and the `'now'` will just add confusion.